src/Command/MigrateOriginalMessageCalls/ConvertOriginalMessageCalls.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2016-2016 Zend Technologies USA Inc. (http://www.zend.com)
```

Comment:
Please fix year. Should be only `2016`, the same in some other new files (12 times)